### PR TITLE
Modified navbar, faq and added a student qoute.

### PR DESCRIPTION
**Pull request by zainvrk**
Added
* give button on navbar
* new faq question and answer
* a student quote

Screenshot 2023-06-08 at 9 57 25 AM
Screenshot 2023-06-08 at 9 58 02 AM

Changed
* footer text
Screenshot 2023-06-08 at 9 58 18 AM

### CODE REVIEW

File: mitx/lms/templates/static_templates/about.html [hunk 1]
```diff
@@ -112,8 +112,23 @@ <h3 class="section-title">Frequently Asked Questions</h3>
...
+                                </p>
+                            </div>
+                        </div>
                     </div>
                 </section>
+                <blockquote class="student-quote">
+                    I cannot overstate the impact that MIT Open Learning has had on my education. I have been a student of MOOCs for several years now, and the courses offered by MIT Open Learning have been some of the most impactful and useful that I have taken. I have been able to take courses on a wide variety of topics, and the quality of the content and the instructors has been consistently excellent. I have also been able to use the courses I have taken to further my career, as many of the topics covered are directly relevant to the work I do. I am extremely grateful to have had the opportunity to take advantage of MIT Open Learning, and I would highly recommend it to anyone who is looking for a high-quality, affordable education.
+                    <span>-Hikmat, Student - College/University, Nepal</span>
```
Comment (ziafazal) [inline mitx/lms/templates/static_templates/about.html]: replace <span> with <div> if it needs to be block level element.

File: mitx/lms/static/sass/views/_about.scss [hunk 2]
```diff
@@ -220,3 +220,39 @@
...
+        position: absolute;
+        left: 10px;
+        top: -10px;
+      }
+
+      &:after {
+        content: '';
+      }
+
+       span {
```
Comment (ziafazal) [inline mitx/lms/static/sass/views/_about.scss]: If we convert <span> to div <div> we can remove some attributes like   `display: block;` and `margin-top: 1em;`

**Review by ziafazal — COMMENTED** [review]
@zainvrk overall looks good if address one minor item.

File: mitx/lms/static/sass/views/_about.scss (hunk 2 above)
Comment (zainvrk) [reply mitx/lms/static/sass/views/_about.scss]: I have tried removing `margin-top`, the gap is not enough.

Without margin-top
Screenshot 2023-06-08 at 2 54 40 PM

With margin-top
Screenshot 2023-06-08 at 2 56 55 PM

**Pull request merged**